packages/react-jsx-highcharts/src/utils/getBoundChartMethod.js

```diff
@@ -0,0 +1,11 @@
+export default function getBoundChartMethod(chart, method, context) {
+  const boundMethod = method.bind(context);
+
+  return function (...rest) {
```

Comment:
Nitpicking, but could we use the convention `...args` here, instead of `...rest`?

`...rest` implies we've destructured some other arguments.